src/app/components/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -1,62 +1,65 @@
 <!-- Table Inline Quick Add -->
-<div class="col-xs-12 col-sm-12 col-md-12 f8-quickadd-table__wiblk"
-  *ngIf="wilistview === 'wi-table-view' || wilistview === 'wi-table-view-top'">
-  <b class="pull-left" style="padding-right: 20px;">{{wilistview === 'wi-table-view'? 'Add Child Work Item' : 'Add Work Item'}}</b>
-  <!-- type dropdown -->
+<div class="col-xs-12 col-sm-12 col-md-12 f8-quickadd__wiblk" #quickAddWidth
+  *ngIf="wilistview === 'wi-table-view' || wilistview === 'wi-table-view-top'"
+  [ngClass]="{'f8-quick-add-inline' : wilistview === 'wi-table-view'}">
   <div class="pull-left">
-    <div class="dropdown-kebab-pf" dropdown>
-      <button class="btn btn-default dropdown-toggle padding-v-0"
-        type="button"
-        dropdownToggle>
-        <i class="fa dib"
-          [ngClass]="selectedType?selectedType.attributes.icon:''"></i>&nbsp;{{selectedType?selectedType.attributes.name:''}}
-        <span class="fa fa-angle-down"></span>
-      </button>
-      <ul class="dropdown-menu"
-        role="menu"
-        aria-labelledby="dropdownKebab"
-        *dropdownMenu>
-        <li *ngFor="let type of allowedWITs"
-            role="menuitem"
-            (click)="selectType($event, type)">
-          <a class="dropdown-item" href="#">
-            <i class="fa pull-left"
-              [ngClass]="type.attributes.icon"></i>&nbsp;{{type.attributes.name}}
-          </a>
-        </li>
-      </ul>
-    </div>
-  </div>
-
-  <div class="f8-quickadd__wiblk-story">
-    <div [ngClass]="wilistview === 'wi-table-view'? 'col-xs-12 col-sm-6 col-md-6' : 'col-xs-12 col-sm-8 col-md-8'"
-      class="f8-quickadd__wiblk-storyinput">
-      <input class="form-control col-xs-12 col-sm-12
-        col-md-12 f8-quickadd-input"
-        type="text"
-        placeholder="Work Item Title"
-        #quickAddTitle
-        [(ngModel)]="workItem.attributes['system.title']"
-        (keyup.enter)="save($event)"
-        (keyup)="checkTitle()"  required/>
+    <b class="pull-left padding-right-15">
+      {{wilistview === 'wi-table-view'? 'Add Child Work Item' : 'Add Work Item'}}
+    </b>
+    <!-- type dropdown -->
+    <div class="pull-left">
+      <div class="dropdown-kebab-pf" dropdown>
+        <button class="btn btn-default dropdown-toggle padding-v-0"
+          type="button"
+          dropdownToggle>
+          <i class="fa dib"
+            [ngClass]="selectedType?selectedType.attributes.icon:''"></i>&nbsp;{{selectedType?selectedType.attributes.name:''}}
```

Comment:
Add a space before and after every ``` : ``` and ``` ? ``` operator. It would be good if you break this line into multiple lines.